### ISSUE TITLE
Browsing images no longer works for select files

### ISSUE THREAD

**Issue by murphysj** [EarthSystemCoG/COG#1051]
WHO: Sylvia

To to add admin_links.png to a wiki page. The URL is not transferred to the image browser. Other images work.

**Comment by murphysj** [comment]
Demoed this to Luca. He was able to reproduce the issue on his system.

**Comment by LucaCinquini** [comment]
This is a really tough problem. I tracked it down to a bad interaction between two third parties components: CK_Editor and django-filebrowser. django-filebrowser must send the image URL to CK_Editor through a Javascript callback, at a specific "window" number. It turns out that the window number is lost IF the search facility in django file browser is used. If not, everything works correctly.

For now, I have inserted a hack in this file:
filebrowser/static/filebrowser/js/FB_CKEditor.js
specifically:
if (CKEditorFuncNum==null || CKEditorFuncNum=='') CKEditorFuncNum=2;

since the number seems to be "2" every time the communication works. Obviously this is not a robust solution, but it seems to be working (hack already deployed on CoG CU server).

If we want to make this more robust, I think we need to upgrade both the CK_Editor and django-filebrowser applications, or replace them with something different. We also need to hire someone who has time to do it...
